lib/Filesystem/Helper.php

```diff
@@ -98,8 +98,7 @@ protected function isBlockable(string $path): bool {
 
 	public function verifyAccess(string $path): bool {
 		$config = OC::$server->getConfig();
```

Comment:
you can also dismiss this creation of $config

Reply:
Oops, nice catch. Done.